packages/checkbox/src/tests/Checkbox.test.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from "react"
 import { render, userEvent, renderHook, invoke } from "@chakra-ui/test-utils"
 import { Checkbox, useCheckbox, CheckboxHookProps } from ".."
+import { CheckboxGroup } from "../CheckboxGroup"
```

Comment:
```suggestion
import { Checkbox, CheckboxGroup, useCheckbox, CheckboxHookProps } from ".."
```

So we can make sure we're testing the public api of the module.

Reply:
Also did `CheckboxGroup` actually make it into a test?

Reply:
Thanks @with-heart, yes, we have tests for `CheckboxGroup`